lib/ansible/modules/utilities/logic/include_tasks.py

```diff
@@ -7,51 +7,53 @@
 __metaclass__ = type
 
 
-ANSIBLE_METADATA = {'metadata_version': '1.1',
-                    'status': ['preview'],
-                    'supported_by': 'core'}
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'core'
+}
 
 
 DOCUMENTATION = '''
 ---
-author:
-    - "Ansible Core Team (@ansible)"
+author: Ansible Core Team (@ansible)
 module: include_tasks
-short_description: dynamically include a task list.
+short_description: Dynamically include a task list
 description:
-     - Includes a file with a list of tasks to be executed in the current playbook.
+  - Includes a file with a list of tasks to be executed in the current playbook.
 version_added: "2.4"
 options:
   free-form:
     description:
-      - This action allows you to specify the name of the file directly w/o any other options.
-      - Unlike M(import_tasks) this action will be affected by most keywords, including loops and conditionals.
+      - The name of the imported file is specified directly without any other option.
+      - Unlike M(import_tasks), most keywords, including loops and conditionals, apply to this statement.
 notes:
-  - This is really not a module, this is a feature of the Ansible Engine, as such it cannot be overridden the same way a module can.
+  - This is really not a module, this is a feature of the Ansible Engine, as such it cannot be overridden the same way
```

Comment:
Please revise to "This is a feature of the Ansible Engine, rather than a module, and cannot be overridden like a module."